tutorials/dataframe/df102_NanoAODDimuonAnalysis.C

```diff
@@ -26,7 +26,7 @@
 #include "TCanvas.h"
 #include "TH1D.h"
 #include "TLatex.h"
-#include "TLorentzVector.h"
+#include "Math/Vector4Dfwd.h"
```

Comment:
You should not be able to survive with a fwd decl. Either the header is misnamed (because it provides the definitions of the templates, not just the fwd decls) or auto-magic auto-heals this, and the header to include here is really the full-blown non-fwd one.